src/API.jl

```diff
@@ -568,9 +568,10 @@ function instantiate(ctx::Context; manifest::Union{Bool, Nothing}=nothing, kwarg
 end
 
 
-status(mode=PKGMODE_PROJECT) = status(Context(), mode)
-function status(ctx::Context, mode=PKGMODE_PROJECT)
-    Pkg.Display.status(ctx, mode)
+status(mode::PackageMode=PKGMODE_PROJECT, pkgs::String...) = status(Context(), mode, pkgs...)
+status(pkgs::String...) = status(PKGMODE_PROJECT, pkgs...)
+function status(ctx::Context, mode::PackageMode=PKGMODE_PROJECT, pkgs::String...)
+    Pkg.Display.status(ctx, Pkg.REPLMode.parse_pkg(String[pkgs...]), mode)
```

Comment:
Is there a better way here?

Reply:
Yes, the parsing is just for the REPL, where we parse the input string. This API function should follow the same pattern as the other API functions and be defined something like
```
status(pkg::Union{String,PackageSpec}) = status([pkg])
status(pkgs::Vector{String}) = status([check_package_name(pkg) for pkg in pkgs])
status(pkgs::Vector{PackageSpec}) = status(Context(), pkgs)
function status(ctx::Context, pkgs::Vector{PackageSpec})
    # implementation
end
```

Reply:
In addition, REPL Mode flags map to API keyword arguments.
```
status(pkg::Union{String,PackageSpec}; mode=PKGMODE_PROJECT) = status([pkg]; mode=mode)
status(pkgs::Vector{String}; mode=PKGMODE_PROJECT) = status([check_package_name(pkg) for pkg in pkgs]; mode=mode)
status(pkgs::Vector{PackageSpec}; mode=PKGMODE_PROJECT) = status(Context(), pkgs; mode=mode)
function status(ctx::Context, pkgs::Vector{PackageSpec}; mode=PKGMODE_PROJECT)
    # implementation
end
```

Reply:
Alright, I'll make the changes.
(Note, however, that `mode` wasn't a keyword argument before this PR either.)